x-pack/plugins/remote_clusters/public/shim.ts

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { npSetup, npStart } from 'ui/new_platform';
+import { management, MANAGEMENT_BREADCRUMB } from 'ui/management';
+import { fatalError } from 'ui/notify';
+import routes from 'ui/routes';
+import { DOC_LINK_VERSION, ELASTIC_WEBSITE_URL } from 'ui/documentation_links';
+
+import { trackUiMetric as track } from '../../../../src/legacy/core_plugins/ui_metric/public';
+
+export function createShim() {
+  const {
+    core: { chrome },
+  } = npSetup;
+
+  const {
+    core: { i18n, notifications, http, injectedMetadata },
+  } = npStart;
+
+  return {
+    coreStart: {
+      chrome,
+      i18n,
+      notifications: {
+        ...notifications,
+        fatalError,
+      },
+      injectedMetadata,
+      http,
+      documentation: {
+        elasticWebsiteUrl: ELASTIC_WEBSITE_URL,
+        docLinkVersion: DOC_LINK_VERSION,
+      },
+      routes: {
+        register: routes.when.bind(routes),
+      },
```

Comment:
@epixa What's the best way for teams to submit requests/proposals for additions or changes to the core API? For example, here I add `fatalError`, `documentation`, and `routes` to the interface. I'd like to bring these suggestions to the awareness of the NP team and other engineers participating in the migration in a central, visible location.

Reply:
Issues or for sufficiently complicated additions an RFC PR. And one of the purposes of the platform consultant thing is so you can just offload that sort of thing directly on that person.